tracing-subscriber/src/layer.rs

```diff
@@ -0,0 +1,420 @@
+use tracing_core::{
+    metadata::Metadata,
+    span,
+    subscriber::{Interest, Subscriber},
+    Event,
+};
+
+use std::any::TypeId;
+
+/// A composable handler for `tracing` events.
+///
+/// The [`Subscriber`] trait in `tracing-core` represents the _complete_ set of
+/// functionality required to consume `tracing` instrumentation. This means that
+/// a single `Subscriber` instance is a self-contained implementation of a
+/// complete strategy for collecting traces; but it _also_ means that the
+/// `Subscriber` trait cannot easily be composed with other `Subscriber`s.
+///
+/// In particular, `Subscriber`s are responsible for generating [span IDs] and
```

Comment:
```suggestion
/// In particular, [`Subscriber`]'s are responsible for generating [span IDs] and
```

Not sure if this is any better but thought we might as well link again?